### PR TITLE
MOSIP-19064: Update all apis with self token rest template

### PR DESCRIPTION
**Pull request by abuabdul**

### CODE REVIEW

File: resident/resident-service/pom.xml [hunk 1]
```diff
@@ -151,6 +151,11 @@
 			<version>${spring.boot.version}</version>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-all</artifactId>
+			<version>4.1.72.Final</version>
+		</dependency>
```
Comment (vishwa-vyom) [inline resident/resident-service/pom.xml:158]: This fix is required only for local rt ? Do we need to commit this ?

File: resident/resident-service/src/main/java/io/mosip/resident/handler/service/SyncAndUploadService.java [hunk 2]
```diff
@@ -177,17 +173,6 @@ public String getFilename() {
...
-			logger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					registartionId,
-					ExceptionUtils.getStackTrace(e));
-			throw new BaseCheckedException(ResidentErrorCode.BASE_EXCEPTION.getErrorCode(), ResidentErrorCode.BASE_EXCEPTION.getErrorMessage(), e);
-
-		} catch (IOException e) {
-			logger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					registartionId,
-					ResidentErrorCode.IO_EXCEPTION.getErrorMessage() + ExceptionUtils.getStackTrace(e));
-			throw new BaseCheckedException(ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorCode(), ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorMessage(), e);
```
Comment (vishwa-vyom) [inline resident/resident-service/src/main/java/io/mosip/resident/handler/service/SyncAndUploadService.java:190]: Why these exceptions are removed ?

File: resident/resident-service/src/main/resources/bootstrap.properties [hunk 3]
```diff
@@ -8,7 +8,7 @@ management.endpoints.web.exposure.include=info,health,refresh
 resident.service=resident
 config.server.file.storage.uri=${spring.cloud.config.uri}/${resident.service}/${spring.profiles.active}/${spring.cloud.config.label}/
 server.port=8099
-server.servlet.path=/resident/v1
+server.servlet.context-path=/resident/v1
```
Comment (vishwa-vyom) [inline resident/resident-service/src/main/resources/bootstrap.properties:11]: Why do we need this change ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/main/resources/bootstrap.properties (hunk 3 above)
Comment (abuabdul) [reply resident/resident-service/src/main/resources/bootstrap.properties:11]: the servlet context path is set in auth adapter. if we dont set it, that goes empty so we are not able to go unauthenticated. Now we removed servlet path added context-path to match it.

File: resident/resident-service/src/main/java/io/mosip/resident/handler/service/SyncAndUploadService.java (hunk 2 above)
Comment (abuabdul) [reply resident/resident-service/src/main/java/io/mosip/resident/handler/service/SyncAndUploadService.java:190]: They are for IO and File Not found exception. They are thrown by tokengenerator. Once i remove the token generator, they are not thrown and i removed it.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**